_data/stage3.yml

```diff
@@ -238,3 +280,13 @@
     - Sven Sauleau
     - Dan Clark
     - Daniel Ehrenberg
+  description: A proposal for syntax to import ES modules with assertions.
+  example: >-
+    new Worker(&#x22;foo.wasm&#x22;, { type: &#x22;module&#x22;, assert: { type:
+    &#x22;webassembly&#x22; } });
+  has_specification: true
+  presented:
+    - date: "September\_2020"
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2020-09/sept-22.md#import-assertions-for-stage-3
```

Comment:
🔧

Reply:
Emojis without context can be an accessibility issue. If you have a comment, please write it out so it is clear for others.